.github/workflows/tests.yml

```diff
@@ -18,21 +22,21 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
 
-      - uses: c-hive/gha-npm-cache@v1
+      - uses: actions/cache@v2
+        with:
+          path: .yarn/global
+          key: yarn
```

Comment:
```suggestion
          key: yarn-${{ hashFiles('yarn.lock') }}
          restore-keys: |
            yarn-
```

Reply:
Thanks!
But wouldn't that make it impossible to share the cache between builds where a dependency changed?

I was kinda aiming for "share the global cache folder between all CI runs", but I might be seeing this perfectly wrong.

Reply:
It will try the exact key first then if there is no match it will take whatever cache was last created

https://github.com/actions/cache/blob/0638051e9af2c23d10bb70fa9beffcad6cff9ce3/README.md#inputs

Reply:
My 2 cents, I would not use 'enableGlobalCache: true', let the recommended cache folder (.yarn/cache) 

I use this action with yarn 3... based on https://github.com/actions/cache/blob/main/examples.md#node---yarn-2

```yml
      # Note: Determine the cache folder(s) to include

      - name: Get yarn cache directory path
        id: yarn-cache-dir-path
        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
      
      - name: Yarn cache
        uses: actions/cache@v2
        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
        with:
          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
          key: yarn-cache-folder-os-${{ runner.os }}-node-${{ env.node-version }}-${{ hashFiles('**/yarn.lock') }}
          restore-keys: |
            yarn-cache-folder-os-${{ runner.os }}-node-${{ env.node-version }}-
            yarn-cache-folder-os-${{ runner.os }}-
```

Reply:
The cache is identical between OS and node version so you don't need to include those in the key

Reply:
Totally, habit I took completely optional.